js/main.js

```diff
@@ -132,7 +148,7 @@ class App extends React.Component {
 
     if (!exists) {
       let stored = JSON.parse(localStorage.getItem(this.keyLS(newPane.id)));
-
+      if (this._bin == null) this.rebin();
```

Comment:
Nit: avoid if statements with single executions following, please use braces even for single statements.